test/Nancy.Tests/Unit/Sessions/CookieBasedSessionsFixture.cs

```diff
@@ -5,12 +5,12 @@ namespace Nancy.Tests.Unit.Sessions
     using System.Linq;
     using System.Reflection;
     using System.Threading;
-    using System.Web;
 
     using FakeItEasy;
 
     using Nancy.Bootstrapper;
     using Nancy.Cryptography;
+    using Nancy.Helpers;

```

Comment:
Are you saying that tests, in this fixture, uses extension methods that came from `System.Web`, but since it's not on Core you added in `Nancy.Helpers` since it had matching extensions that could be used instead?